### DISCUSSION TITLE
Date formats for formula [Jun 29]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6022] [Discord Threads]
> 🙋‍♂️ **bookarobanzai**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### formatting formula output explicitly is currently not supported via UI  
#  
> 🙋‍♂️ **bookarobanzai**  
#### Thank's for you quicky response.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1112842005190557788)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
